packages/devtools_app/lib/src/charts/chart_controller.dart

```diff
@@ -478,20 +481,20 @@ class ChartController extends DisposableController
     return traceIndex;
   }
 
-  int get numberOfVisibleXAxisTicks => visibleTicks;
+  int? get numberOfVisibleXAxisTicks => visibleTicks;
 
   /// If negative then total ticks collected < number of visible ticks to display.
-  int get totalTimestampTicks => timestampsLength - numberOfVisibleXAxisTicks;
+  int get totalTimestampTicks => timestampsLength - numberOfVisibleXAxisTicks!;
 
   int get leftVisibleIndex {
     final leftIndex = totalTimestampTicks;
     if (leftIndex > 0) return leftIndex;
 
     // Less ticks than total size of ticks to show.
-    return numberOfVisibleXAxisTicks - timestampsLength;
+    return numberOfVisibleXAxisTicks! - timestampsLength;
   }
 
-  bool isTimestampVisible(int timestamp, int timestampIndex) {
+  bool isTimestampVisible(int? timestamp, int timestampIndex) {
```

Comment:
remove the timestamp parameter completely as it is not used

Reply:
done